node.js

```diff
@@ -10,3 +10,80 @@ document.querySelectorAll('ul.ul-nav > li').forEach((n) => n.addEventListener('c
   navButton2.classList.remove('active');
   navMenu.classList.remove('active');
 }));
+
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly make sure the projects section is created dynamically with the information from the projects array. Modal popup data is populated with the data from the projects array.